pinot-query-planner/src/test/java/org/apache/pinot/query/QueryEnvironmentTestBase.java

```diff
@@ -130,9 +131,16 @@ protected Object[][] provideQueries() {
 
   public static QueryEnvironment getQueryEnvironment(int reducerPort, int port1, int port2,
       Map<String, Schema> schemaMap, Map<String, List<String>> segmentMap1, Map<String, List<String>> segmentMap2) {
+    return getQueryEnvironment(reducerPort, port1, port2, schemaMap, segmentMap1, segmentMap2, Collections.emptyMap());
+  }
+
+  public static QueryEnvironment getQueryEnvironment(int reducerPort, int port1, int port2,
+      Map<String, Schema> schemaMap, Map<String, List<String>> segmentMap1, Map<String, List<String>> segmentMap2,
+      Map<String, Boolean> nullHandlingMap) {
     MockRoutingManagerFactory factory = new MockRoutingManagerFactory(port1, port2);
     for (Map.Entry<String, Schema> entry : schemaMap.entrySet()) {
-      factory.registerTable(entry.getValue(), entry.getKey());
+      // ALWAYS enable null handling in V2.
+      factory.registerTable(entry.getValue(), entry.getKey(), nullHandlingMap.getOrDefault(entry.getKey(), false));
```

Comment:
Does this need to be: `nullHandlingMap.getOrDefault(entry.getKey(), true)`?

Reply:
no. by default nullability is disabled. this is the default behavior of all tables for pinot nowadays.

Reply:
The comment was wrt `// ALWAYS enable null handling in V2.`

Reply:
ah i think the comment was not suppose to be there